templates/ts/custom-copilot-basic/src/prompts/chat/skprompt.txt

```diff
@@ -0,0 +1 @@
+You are an AI bot that can chat with users.
```

Comment:
[Minor] @kuojianlu We are going to reuse this template for Custom API template, and we need to update this file. We will overwrite config.json and skprompt for update/add some description from OpenAPI spec. Please let me know if you have any better method for updating these files.